test/Docfx.MarkdigEngine.Extensions.Tests/PlantUmlTest.cs

```diff
@@ -7,7 +7,7 @@ namespace Docfx.MarkdigEngine.Tests;
 
 public class PlantUmlTest
 {
-    [Fact]
+    [Fact(Skip = "Unstable test")]
```

Comment:
instead of skipping, can we relax the assertion to say that we expect the result to start with `<div class="lang-plantUml"><?xml version="1.0" encoding="us-ascii" standalone="no"?><svg` and end with `</svg></div>`?

Reply:
OK. I'll trying to fix assertions.

Reply:
It seems a bit complicated to reuse existing `TestUtility.VerifyMarkup` to test startWith/endsWith.

How about temporary merge this PR.
And revert changes after the PlantUML Online server response is stabilized?

Reply:
Make `VerifyMarkup` return `actualHtml` and pass `html: null` and verify the result?

Reply:
If it's acceptable to change existing `VerifyMarkup` method signature.
It can easily verify markup result,

I've created commit by https://github.com/dotnet/docfx/pull/9858/commits/ac73209c5bd111f7d064ad66547ee10db865390d